File: src/mispobjects/eventtag.go
package mispobjects

type EventTag struct {
	Id             string `json:"id"`
	Name           string `json:"name"`
	Colour         string `json:"colour"`
	Exportable     bool   `json:"exportable"`
	OrgId          string `json:"org_id"`
	UserId         string `json:"user_id"`
	HideTag        bool   `json:"hide_tag"`
	NumericalValue string `json:"numerical_value"`
	Local          bool   `json:"local,omitempty"`
	EventId        bool   `json:"event_id,omitempty"`
	TagId          string `json:"tag_id"`
	Tag            Tag    `json:"Tag"`
}

File: src/main/kotlin/com/devo/feeds/data/misp/EventTag.kt
package com.devo.feeds.data.misp

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class EventTag(
    val id: String? = null,
    val name: String? = null,
    val colour: String? = null,
    val exportable: Boolean? = null,
    @SerialName("org_id") val orgId: String? = null,
    @SerialName("user_id") val userId: String? = null,
    @SerialName("hide_tag") val hideTag: Boolean? = null,
    @SerialName("numerical_value") val numericalValue: String? = null,
    val local: Boolean? = null,
    @SerialName("event_id") val eventId: Boolean? = null,
    @SerialName("tag_id") val tagId: String? = null,
    @SerialName("Tag") val tag: Tag? = null
)
